Cisco/cisco-meraki/tests/test_flows_allow_ipv6.json

```diff
@@ -0,0 +1,53 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco Meraki",
+        "dialect_uuid": "995d7daf-4e4a-42ec-b90d-9af2f7be7019"
+      }
+    },
+    "message": "1673277220.253011885 FW_MX_01 flows src=1.2.3.4 dst=5.6.7.8 protocol=tcp sport=39247 dport=443 pattern: 0 (tcp || udp) && dst port 443"
```

Comment:
The filename is suffixed with `ipv6`. However, I see only ipv4 in the message. Is a mistake?